layouts/_default/list.html

```diff
@@ -1,10 +1,9 @@
 {{ define "title" }}
   {{- if eq .Kind "term" -}}
-    {{- i18n .Data.Singular | title -}}
+    {{- i18n .Data.Plural 1 | title -}}
```

Comment:
Is the`1` intentional? I couldn't find about it in the Hugo's doc.

Reply:
Got it. It is the value to pick the correct translation. Saw it [here](https://gohugo.io/content-management/multilingual/#query-a-flexible-translation-with-variables).

Reply:
Yes exactly. :)